languages/ruby/README.md

```diff
@@ -96,7 +96,18 @@ puts response
 
 ## Development
 
+Prerequisites:
+
+- Ruby >= 3.0 installed
+- Generate schemas `npm run schemas`
+
 ```bash
+# Navigate to the ruby language folder
+cd languages/ruby
+
+# Make the binary folder if it doesn't exist already
+mkdir ./bitwarden_sdk_secrets/lib/macos-arm64
```

Comment:
I think we should change the command to make it create the parent directories if they are missing, and not output an error if the folder exists:
```suggestion
mkdir -p ./bitwarden_sdk_secrets/lib/macos-arm64
```

Reply:
Great idea added in https://github.com/bitwarden/sdk/pull/720/commits/d597a11b60ff22eeb11bd49ca9ce35a64e377857